libstuff/libstuff.cpp

```diff
@@ -2621,6 +2621,9 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
         // We should always avoid logging authTokens because they give access to accounts
         pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("\"authToken\":<REDACTED>", &sqlToLog);
 
+        // Let's redact queries that contain encrypted fields since there's no value in logging them
+        pcrecpp::RE("v[0-9]+:[0-9A-F]{10,}").GlobalReplace("<REDACTED>", &sqlToLog);
```

Comment:
Why 10?

Reply:
kind of arbitrary, since all the encrypted stuff have a minimum length (not sure what that is) but it's at least 10 I think.

Reply:
I'll approve this but should we make a followup issue to move this to Auth at some point? It's one of those things that might never get done, I guess. Bedrock really is not the right place for this, though.